### DISCUSSION TITLE
Scrolling not working

### DISCUSSION THREAD

**Discussion by hawswebmaster** [code4recovery/12-step-meeting-list#1140] [Q&A] [answered]
Hi,

New here and new to basic plugin knowledge,

In the state scroll down box, members on safari/mac cannot scroll down. I will post pictures. Can anyone help please?
![image](https://github.com/code4recovery/12-step-meeting-list/assets/134229710/65bfaeda-eebc-489c-bd09-acfeaf61a6ec)

Also, the search seems inconsistent, when typing in "Texas" the search is not pulling from all of the meetings and says no meetings were found even though there are plenty of Texas meetings. Is there any way for the search bar to pull from all of the meeting information?

![image](https://github.com/code4recovery/12-step-meeting-list/assets/134229710/6b8a1200-5371-4d8f-8a76-3915437d5ca5)

url is https://heroinanonymous.org/meetings/

**Comment by kiyote33** [comment]
Hi @hawswebmaster,
Hopefully I can help you out here.  On your first question I don't have a Mac to test, but I believe the default behavior for the mouse scroll may be to scroll the entire page. If that's what is happening, can you try scrolling down your region list using the down arrow key? I'm not sure if we can implement a fix for that, but we will research it and see if there's a solution.
Your second question deals with our search feature on the meeting list page, which is designed to help find a specific meeting(s) by name or group. So, if you had a meeting named Texas it would search the meeting "name" or "group" columns in the database looking to return a record subset of all records that included the word 'Texas'. The reason you get the "No meetings..." message is that we don't search the Regions field. Your website has done a nice job of setting up the Region dropdown to return all the meetings in Texas. This takes us back to your first question which I think if it gets resolved will solve both issues. I will put a query out in our developers channel to see if anyone knows of any other work-arounds for the Mac not scrolling issue.

**Reply by hawswebmaster** [reply]
Thank you

**Reply by hawswebmaster** [reply]
Also, the two members that are working with me on this, as I do not have a mac either, said that using the arrow key did not work.

**Reply by kiyote33** [reply]
Feedback from our developer community suggests it might be a theme specific issue. Could you try switching themes (maybe try one of WordPress's default Twenty themes) and see if the issue persists? Thx...

**Reply by hawswebmaster** [reply]
We have switched the theme on a blog site and the problem is still going on with the scrolling, mac/safari. 

https://www.heroinanonymous.org/blog/meetings1/

As to my other concern, is there a way to make the search criteria include all data? Not just meeting name or group?

**Reply by kiyote33** [reply]
Hi again @hawswebmaster,
Yesterday I indicated I don't have access to a Mac desktop, but I did manage to dig up an old IPad which shows scrolling working correctly for the Regions dropdown on https://heroninanonymous.org/meetings on multiple browsers including Safarii and Edge.. A developer with access to a Mac confirmed they found the same thing when they checked it. Maybe you could get it rechecked to see if the issue has gone away for you also.
Your request to expand the search criteria seems valid so we will initiate an issue ticket for the dev community to look into expanding the search criteria to other fields.

**Reply by kiyote33** [reply]
@hawswebmaster If you would like to send us the authorized link for your site we could try loading your meetings in a test environment and possibly give you a better solution if the problem is persisting for you?

**Reply by hawswebmaster** [reply]
We will be trying it again on an ipad later today. I am not sure about an authorized link, but i do have the meeting csv. would that help on your test site?

**Reply by kiyote33** [reply]
Absolutely. I'll try it in an environment that mimics your setup. What theme are you using? Do you have many plugins?

**Reply by kiyote33** [reply]
You can email me the file at waynep@code4recovery.org if you like.

**Reply by hawswebmaster** [reply]
25 active plugins, Bodega theme 3.4, WP 6.2, Apache, PHP 7.4.33

I will email the file.

**Reply by kiyote33** [reply]
Hi again @hawswebmaster, not a lot of success in replicating your issue in a pristine environment (built with WP 6.2.2, Apache, PHP 7.4.30, and a MariaDB 10.4.10 database). Unable to get the Bodega theme (3.4) from Envato Market as it is a premium product, but did get a couple of freebees from them (Tulip & Yoome). All my testing in this pristine setting using the following free themes did not replicate our issue: Astra, Storefront Mania, Th Shop Mania, Twenty Ten, Twenty Fourteen, Twenty TwentyThree, and the two themes from Envato Market. All worked perfectly (except the Yoome theme which had some unrelated CSS display issues). The only plugin I've loaded besides our **12 Step Meeting List** is the latest free version of W3 Total Cache which I see you are using. If you could provide a screen shot here of the rest of your active plugins, I could try installing them to see if one of them is in conflict.

**Reply by kiyote33** [reply, marked as answer]
So I was able to replicate the issue as soon as the Bodega theme was installed, before installing any of the other plugins. As I don't have a license for that theme, can you advise if there are any updates available? I went ahead and installed all of the required plugins for the Bodega theme and there wasn't any change on the behavior of the dropdown regions list scrolling. When testing against all the other themes, they behaved correctly which confirmed my thinking that it's the Bodega theme causing the problem. Looking at the Developer Console on the meetings page, I'm seeing this error: "**bodega.test/wp-content/themes/bodega/img/logo.png:1     Failed to load resource: net::ERR_NAME_NOT_RESOLVED**". Looks like some theme developer code that should have been removed before releasing the theme. I would think you should contact the Envato Market team to get a patch or fix for this issue.  Hope this helps resolve your issue.

**Reply by hawswebmaster** [reply]
Thank you very much. I have reached out to the theme creator about this issue. I will keep an eye on the request for the search expansion ticket. I really appreciate all your help. I will mark this as answered.
